package.json

```diff
@@ -10,11 +13,13 @@
     "preinstall": "cd generator-theia && npm install",
     "prepare": "npm install generator-theia && npm run bootstrap && npm run clean && npm run build",
     "prepublishOnly": "npm run test",
-    "bootstrap": "node scripts/generate.js && lerna bootstrap --hoist",
+    "bootstrap": "node scripts/generate.js && lerna bootstrap --hoist -- --no-optional",
```

Comment:
https://github.com/trentm/node-bunyan/blob/master/package.json#L25 it declares optional dependencies here